### PR TITLE
Abacard stat counter

### LINKED ISSUE #1079: Abacard stats counter/view
It would be nice with a small counter/view with the stats about the attendees, so they don't have to switch to the administrate view to check the list.

### PR DESCRIPTION
**Pull request by amalieeh**
fix #1079 Lets users see the stats about the attendees.

before:
image

after:
image

### CODE REVIEW

File: app/routes/events/components/EventAdministrate/Abacard.js [hunk 1]
```diff
@@ -79,6 +85,10 @@ class Abacard extends React.Component<*, State> {
             this.input = input;
           }}
         />
+        <div className={styles.counter}>
+          {' '}
```
Comment (ekmartin) [inline app/routes/events/components/EventAdministrate/Abacard.js]: do you need this empty space or did prettier just put it in?

**Comment by ekmartin** [comment]
nice! this would be good to have on the `Påmeldinger` page as well (the other admin tab). we should merge this first though :)

File: app/routes/events/components/EventAdministrate/Abacard.js (hunk 1 above)
Comment (amalieeh) [reply app/routes/events/components/EventAdministrate/Abacard.js]: Prettier put it in. Should I remove it?

Comment (ekmartin) [reply app/routes/events/components/EventAdministrate/Abacard.js]: yeah, I don't think you need it (it's on its own line anyway right, so a space shouldn't matter)

**Pull request merged**